compiler/resolution/wrappers.cpp

```diff
@@ -718,6 +719,11 @@ static Symbol* createDefaultedActual(FnSymbol*  fn,
     newCall->insertAtTail(new SymExpr(mapTo));
   }
 
+  // If the new call throws and it was in a 'try!' before
+  // we moved it, put it into a new 'try!'
+  if (throws && call->tryTag == TRY_TAG_IN_TRYBANG)
+    newCall = new CallExpr(PRIM_TRYBANG_EXPR, newCall);
```

Comment:
I don't already have example that behaves differently (but maybe you can come up with one) but I would expect this to also handle TRY_TAG_IN_TRY by putting it into PRIM_TRY_EXPR. This might only matter in non-standard error handling modes (e.g. if the module had `pragma "error mode strict"`) where calls that could throw out of a function must be marked.

Reply:
Thanks! I just checked in a test for `try` and the change to make it work. I'll do another paratest, but I think it's good.